h2o-core/src/test/java/hex/CreateFrameTest.java

```diff
@@ -21,12 +23,16 @@
     cf.positive_response = false;
     cf.has_response = true;
     cf.seed = 1234;
-    Frame frame = cf.execImpl().get();
-    Assert.assertTrue(frame.numCols() == 11);
-    Assert.assertTrue(frame.numRows() == 100);
-    // Tries to print a frame
-    frame.toString();
-    frame.delete();
+    Frame frame = null;
```

Comment:
To avoid hiding errors of incorrect usage of Scope in the actual tested code. If CreateFrame used Scope.track to delete any temporary data, the test will not detect any leakage, while the delete approach will.

Since I am making this change because stuff was leaking from this test, I want to be careful about deleting only stuff that is created by the test.